boards/seeeduino_XIAO/board.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C)  2021 Franz Freitag, Justus Krebs, Nick Weiler 
```

Comment:
```suggestion
 * Copyright (C)  2021 Franz Freitag, Justus Krebs, Nick Weiler
```

(remove trailing whitespace)